src/content/docs/new-relic-solutions/best-practices-guides/alerts-applied-intelligence/alerts-best-practices.mdx

```diff
@@ -16,36 +16,86 @@ redirects:
+
+## Notification channels [#channel-practices]
+
+Tailor notifications to the most useful channel and policy so you can avoid alert fatigue and help the right personnel receive and respond to incidents they care about in a systematic way.
+
+<Callout variant="tip">
+Learn how to [set up notification channels](/docs/alerts-applied-intelligence/new-relic-alerts/alert-notifications/notification-channels-control-where-send-alerts/) if you’re new to Alerts.
+</Callout>
+
+Notify teams and individuals who:
+- Can resolve a problem when an incident arises
+- Need to stay updated when an incident arises
+
+For staying updated, select a notification channel that is less intrusive like email.
 
-## Control how many notifications you get [#more-or-less]
+For vital notifications and responding to the incident, select a notification channel that is immediate and reliable like PagerDuty or Slack. You are given a lot more control over who gets notified and when with PagerDuty and OpsGenie. 
 
-The more [conditions](/docs/alerts/new-relic-alerts-beta/configuring-alert-policies/define-alert-conditions) you define, the more [incidents](/docs/alerts/new-relic-alerts-beta/reviewing-alert-incidents/explore-incident-history) can be triggered and monitored. For example, your organization may need an alerting solution to accommodate extensive IT systems. Create policies with multiple conditions for [multiple monitored entities](/docs/alerts/new-relic-alerts-beta/configuring-alert-policies/select-product-targets-alert-condition) that notify you through one or more notification channels. Set [incident preference](/docs/alerts/new-relic-alerts/configuring-alert-policies/specify-when-new-relic-creates-incidents) to determine how violations lead to notifications.
+Do not rely on email for quick notifications in case of mail delays. An email list will also bounce if one is no longer valid.
 
-On the other hand, your organization may not need an extensive alerting structure. The fewer conditions you define and the more minimal your incident preference is, the fewer violations will be opened. For example, for a simple alerting solution, you could create a simple policy with only an email [notification channel](/docs/alerts/new-relic-alerts/managing-notification-channels/notification-channels-controlling-where-send-alerts).
 
-## Set thresholds for conditions [#conditions]
+## Incident preferences [#incident-practices]
 
-Set the [thresholds](/docs/alerts/new-relic-alerts-beta/configuring-alert-policies/define-thresholds-trigger-alert) for your policy's conditions to meaningful levels for your environment. Here are some suggested guidelines:
+Decide when you get incident notifications so you can respond to incidents when they happen.
+
+<Callout variant="tip">
+Learn more about your [incident preferences options](https://discuss.newrelic.com/t/relic-solution-alert-incident-preferences-are-the-key-to-consistent-alert-notifications/40867) if you’re new to Alerts.
+</Callout>
+
```

Comment:
"Change your default incident preference setting to better understand the exact scope and number of all violations." I'm not clear on how changing my default incident preference will help me better understand. 

I feel like there's something in between the two halves of this sentence. A missing logical piece.

1. Change default preferences
2. ???
3. Better understand 

See? :)